### DISCUSSION TITLE
Welcome to Community.VisualStudio.VSCT Discussions!

### DISCUSSION THREAD

**Discussion by madskristensen** [VsixCommunity/Community.VisualStudio.VSCT#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by CZEMacLeod** [comment]
Hey all. I'm a dotnet dev, who has been working with various Microsoft dev environments since the days of Access 1.0. I started writing add-ins in the VB6/VBA days. 
Now work with VB.Net and C#. I code a lot of MSBuild Tasks and MSBuild logic for my company's internal processes/products, and also have built the occasional internal extension. 
@madskristensen's extensions are a life-saver in many cases to make the environment so much nicer to use - and so I felt that, where possible, I should give back to the community when I can. Especially where collaborative/open development has helped me solve an issue which I have implemented for internal use, but is generic enough that it could be of use to others.
